nixos-modules/microvm/mounts.nix

```
@@ -87,7 +87,8 @@ lib.mkIf config.microvm.guest.enable {
           upperdir = "${writableStoreOverlay}/store";
           workdir = "${writableStoreOverlay}/work";
         };
-        options = lib.optional isRwStoreVirtiofsShare "userxattr";
+      } // lib.optionalAttrs isRwStoreVirtiofsShare {
+        options = [ "userxattr" ];
```

```suggestion
        options = lib.mkIf isRwStoreVirtiofsShare [ "userxattr" ];
```
We should really use this 😅